packages/cli/src/UserManagement/email/UserManagementMailer.ts

```diff
@@ -83,48 +90,85 @@ export class UserManagementMailer {
 	}
 
 	async notifyWorkflowShared({
-		recipientEmails,
-		workflowName,
+		sharer,
+		newShareeIds,
+		workflow,
 		baseUrl,
-		workflowId,
-		sharerFirstName,
 	}: {
-		recipientEmails: string[];
-		workflowName: string;
+		sharer: { id: string; firstName: string };
+		newShareeIds: string[];
+		workflow: { id: string; name: string };
 		baseUrl: string;
-		workflowId: string;
-		sharerFirstName: string;
 	}) {
+		if (this.isEmailSetUp) return;
```

Comment:
I we change this to `if (!this.mailer) return;`, then we can remove the `?.` on line 113.